release-notes.md

```diff
@@ -1,3 +1,14 @@
+# Release 4.0.0
+- [#55](https://github.com/linq2db/linq2db.LINQPad/pull/65) Fix bug in pluralization/singularization code
+- [ALL] dependency update: linq2db 3.6.0 -> 4.0.1
+- [ALL] dependency update: LINQPAD.Reference 1.1.0 -> 1.3.0
+- [ALL] dependency update: Humanizer.Core 2.13.14 -> 2.14.1
+- [ALL] dependency update: CodeJam 4.0.1 -> 4.1.0
+- [ALL] dependency update: System.Memory 4.5.4 -> 4.5.5
+- [ALL] dependency update: MySqlConnector 2.0.0 -> 2.1.10
+- [ALL] dependency update: Oracle.ManagedDataAccess.Core 3.21.4 -> 3.21.61
+- [LINQPAD7] dependency update: Oracle.ManagedDataAccess 19.13.0 -> 21.6.1
```

Comment:
add reference to https://github.com/linq2db/linq2db.LINQPad/issues/63

Reply:
Ah, also check affected LINQPAD version for dependencies. I don't think all of them updated for `ALL`